test/test_jit.py

```diff
@@ -3297,23 +3297,29 @@ def test_print_op_module(self):
 
     def test_serialize_qtensor(self):
         class SimpleQTensor(torch.jit.ScriptModule):
-            def __init__(self):
+            def __init__(self, per_channel):
                 super(SimpleQTensor, self).__init__()
                 x = torch.rand(5, 5).float()
-                x_q = torch.quantize_linear(x, 0.2, 10, torch.quint8)
+                if not per_channel:
+                    x_q = torch.quantize_linear(x, 0.2, 10, torch.quint8)
+                else:
+                    s = torch.rand(5, dtype=torch.float64) + 0.1
+                    zp = torch.randint(5, 15, (5,))
+                    x_q = torch.quantize_linear_per_channel(x, s, zp, [1], torch.quint8)
 
                 self.register_buffer('x', x_q)
 
             @torch.jit.script_method
             def forward(self):
                 return self.x
 
-        model = SimpleQTensor()
-        buffer = io.BytesIO()
-        torch.jit.save(model, buffer)
-        buffer.seek(0)
-        model_loaded = torch.jit.load(buffer)
-        self.assertEqual(model_loaded(), model())
+        for per_channel in [False, True]:
```

Comment:
Can we also sweep over qschemes in this test?

Reply:
@raghuramank100 - it's already sweeping among the two qschemes we support. We don't currently pass qscheme to any API and there are two calls quantize_linear and quantize_linear_per_channel to create tensor, so I don't think we can create symmetric tensor today